### ISSUE TITLE
Add code examples to estimation api docs

### ISSUE THREAD

**Issue by jo11he** [tudat-team/tudat-multidoc#30]
I have not yet included the code-examples section in the estimation API docs, where we started to put code snippet examples for the usage of each documented function (see environment_setup, propagation_setup API docs for example).

@DominicDirkx you can probably assign somebody who is most suited for this task.

**Comment by larshinueber** [comment]
Hi @jo11he,
It seems that we now have code examples for most of the functionalities in `estimation_setup` and `estimation_setup.observation`, but missing examples for `estimation`.
Without having worked a lot with the estimation part of tudat, should we also add code examples for `estimation` or is this already covered in the example applications? In that case I would close this issue and add anything that is missing code examples to https://github.com/tudat-team/tudatpy/issues/211

**Comment by DominicDirkx** [comment]
Yep, this indeed also needs some code examples. For entries where we only need to add examples, could we open a separate issue to track these (with a similar list as in 211)?
